ui/shared/components/panel/family/FamilyFields.jsx

```diff
@@ -65,12 +65,12 @@ BaseAnalystEmailDropdown.propTypes = {
 
 const mapDropdownStateToProps = state => ({
   loading: getUserOptionsIsLoading(state),
-  options: getAnalystOptions(state),
+  options: getUserOptions(state),
 })
 
-const mapDropdownDispatchToProps = {
-  load: loadAnalystOptions,
-}
+const mapDropdownDispatchToProps = (dispatch, ownProps) => ({
+  load: () => dispatch(loadUserOptions(ownProps.meta.form.split('_-_')[1])),
```

Comment:
The delimitation is a little weird. Does the double under score `__` work?

Reply:
this is how we do it everywhere else in the code base,  so it is a) more consistant and b) isafer because some family IDs do have a double underscore

Reply:
Okay.